docs/specs/bug_and_feedback_submission.md

```diff
@@ -48,9 +48,40 @@ When putting an object into a bucket [the client must give it a key](https://doc
 
 This opens a possible attack vector for incurring large bills to SatoshiLabs by an attacker uploading large amounts of data to the buckets. This can't be 100% prevented as the API must remain open because Suite's users don't authenticate in any way. A rate limiting could prevent this but that would require some kind of application layer in between the clients and the S3. Instead, for start, we'll start by setting up a [CloudWatch alarm](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html) that will warn us if daily uploads exceed some threshold.
 
+## Data
+This section describes what data are sent in the query string. We start with smaller set of fields that can all fit into one level of nesting so that we don't have to decide [which encoding convention](https://stackoverflow.com/a/9547490/5698865) to use for nested fields. More 
+
+### Bugs
+- `description` The textual description of the bug written by the user.
+- `path` Current url path. Corresponds to the "category" in the Designs.
+- `screenshot` (optional) UUID of linked image.
+
+### Feedback
+- `rating` 1-5, 1 is best, 5 is worst.
+- `description` (optional) User's impressions of the app.
+- `screenshot` (optional) UUID of linked image.
+
+### Common
+The fields sent with both bugs and feedback. All device-related fields are optional because the can be no device connected. Most of them can be reused from the [GitHub Issue feature](https://github.com/trezor/trezor-suite/blob/78736782fbd6556f536fcbc989ba15cf0b367555/packages/suite/src/services/github.ts#L36).
+- `platform` eg. `web` or `desktop`
+- `os`: eg. `windows` or `mac`
+- `user_agent` the user agent string
+- `suite_version` eg. `21.6.0`
+- `suite_revision` commit sha of the build of Suite
+- `screen_dimensions` eg. `19250x1080`
+- `device_type` (optional) eg. `T` or `One`
+- `firmware_version` (optional) versions
+- `firmware_revision` (optional) commit sha of the build of FW
+- `firmware-type` eg. `bitcoin-only` or `regular`
```

Comment:
Maybe it is a crazy idea but I was considering to have a field called _markdown_ which would contain these fields and as well the bug description (again inspiration [here](https://github.com/trezor/trezor-suite/blob/78736782fbd6556f536fcbc989ba15cf0b367555/packages/suite/src/services/github.ts#L36)). This way Alex could move the bug report from the Google Sheet (or wherever) really simply by just copy & pasting this field to Github.

Reply:
I'd rather do that in Keboola or whichever tool the data ends up in. Mainly to reduce duplication and to not transmit and store more data than necessary.

OTOH the issue template is already present in Suite's code so we might just easily reuse it there. [Added](7725d0715bfb7f575001a3a4f05542c3e8a4be24) the field to the Bug fields as it IMO doesn't make much sense to have the template in feedback fields.